src/transformers/models/superpoint/image_processing_superpoint.py

```diff
@@ -270,3 +274,46 @@ def preprocess(
         data = {"pixel_values": images}
 
         return BatchFeature(data=data, tensor_type=return_tensors)
+
+    def post_process_keypoint_detection(
+        self, outputs: SuperPointKeypointDescriptionOutput, target_sizes: torch.Tensor
+    ):
+        """
+        Converts the raw output of [`SuperPointForKeypointDetection`] into lists of keypoints, scores and descriptors
+        with coordinates absolute to the original image sizes.
+
+        Args:
+            outputs ([`SuperPointKeypointDescriptionOutput`]):
+                Raw outputs of the model.
+            target_sizes (`torch.Tensor` of shape `(batch_size, 2)`):
```

Comment:
small nit: I think we should also give an option to have this as a tuple of tuples of same size like in OWL to save users of the of converting it to tensor everytime and check inside whether it's a tensor or not
I'd imagine user passing in `[image.size for image in images]` the easiest

Reply:
You are right, I also thought about it in the first place. I've added this option